docs/content/en/docs/clustermgmt/cluster-backup-restore/restore-cluster.md

```diff
@@ -0,0 +1,245 @@
+
+    # The clusterctl move command needs to be executed for each workload cluster.
+    # It will only move the workload cluster resources from the EKS Anywhere backup to the new management cluster.
+    # If you have multiple workload clusters, you have to run the command for each cluster as shown below.
+
+    # Move workload cluster w01 resources to the new management cluster mgmt-new
+    docker run -i --network host -w $(pwd) -v $(pwd):/$(pwd) --entrypoint clusterctl ${CLI_TOOLS_IMAGE} move \
+        --namespace eksa-system \
+        --filter-cluster {WORKLOAD_CLUSTER_1} \
+        --from-directory ${CLUSTER_STATE_BACKUP_LATEST_PATH} \
+        --to-kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    
+    # Move workload cluster w02 resources to the new management cluster mgmt-new
+    docker run -i --network host -w $(pwd) -v $(pwd):/$(pwd) --entrypoint clusterctl ${CLI_TOOLS_IMAGE} move \
+        --namespace eksa-system \
+        --filter-cluster {WORKLOAD_CLUSTER_2} \
+        --from-directory ${CLUSTER_STATE_BACKUP_LATEST_PATH} \
+        --to-kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    ```
+
+1. (Optional) Update the cluster config file of the workload clusters if the new management cluster has a different cluster name than the original management cluster.
+
+    You can **skip this step** if the new management cluster has the same cluster name as the old management cluster.
+
+    ```yaml
+    # workload cluster w01
+    ---
+    apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+    kind: Cluster
+    metadata:
+      name: w01
+      namespace: default
+    spec:
+      managementCluster:
+        name: mgmt-new # This needs to be updated with the new management cluster name.
+      ...
+    ```
+
+    ```yaml
+    # workload cluster w02
+    ---
+    apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+    kind: Cluster
+    metadata:
+      name: w02
+      namespace: default
+    spec:
+      managementCluster:
+        name: mgmt-new # This needs to be updated with the new management cluster name.
+      ...
+    ```
+
+    Make sure that apart from the `managementCluster` field you updated above, all the other cluster configs of the workload clusters need to stay the same as the old workload clusters resources after the old management cluster fails.
+
+    Apply the updated cluster config of each workload cluster in the new management cluster.
+
+    ```bash
+    MGMT_CLUSTER_NEW="mgmt-new"
+    MGMT_CLUSTER_NEW_KUBECONFIG=${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
+
+    kubectl apply -f w01/w01-eks-a-cluster.yaml --kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    kubectl apply -f w02/w02-eks-a-cluster.yaml --kubeconfig ${MGMT_CLUSTER_NEW_KUBECONFIG}
+    ```
+
+1. Validate all clusters are in the desired state.
+
+    ```bash
+    kubectl get clusters -n default --kubeconfig ${MGMT_CLUSTER_NEW}/${MGMT_CLUSTER_NEW}-eks-a-cluster.kubeconfig
```

Comment:
you could check here that the eks-a clusters are also ready through their status
like with a custom column in this command